set.mm

```diff
@@ -534636,6 +534629,37 @@ extends this idea to multiple additions (TODO).
       KLMNO $.
   $}
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Related to ax6v
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  ${
+    $d x y $.
+    wl-speiv.1 $e |- ( x = y -> ( ps -> ph ) ) $.
+    wl-speiv.2 $e |- ps $.
+    $( Theorem ~ speiv is not minimalistic in the sense it uses only necessary
+       hypotheses.  The first can be simplified to an implication, so ~ speiv
+       specializes this one for convenience, similar to ~ spvv vs. ~ spimw .
+       (Contributed by Wolf Lammen, 22-Oct-2023.) $)
+    wl-speiv $p |- E. x ph $=
+      ( wex hbth spimew ax-mp ) BACGFBACDBCFHEIJ $.
```

Comment:
Indeed. I was wondering the same thing after your previous PR. I wrote speiv with a biconditional in the hypothesis to mimic ~spei, and because it is the general idiom for implicit substitution (even if here, there are no associated DVs, so it's more general).

Maybe you can add this (in addition to speiv, so find another label), and similarly add the strengthened version of spei. And then prove exgen as you do in wl-exgen.

Reply:
Thanks for speiv/speivv. Actually, maybe a better label for speivv would be speivw since it's weaker than speiv but has no additional DV condition (sorry I didn't come up with this label sooner).
Can you do the same with spei ? (that is, relabel it speiw and add the stronger form as spei) ?